src/bootman/kernel.c

```diff
@@ -206,7 +206,19 @@ Kernel *boot_manager_inspect_kernel(BootManager *self, char *path)
                 buf = NULL;
         }
 
+        /* Use kernel's command line and /etc/kernel/cmdline if present */
+        if (self->cmdline) {
+                char *cm = NULL;
+                if (asprintf(&cm, "%s %s", kern->cmdline, self->cmdline) < 0) {
```

Comment:
From what I can see the kernel consumes special parameters like `root=`. Later assignments can be used to break the 10 value limit, but I don't know how this works for root=, whether its an override, ignore or append .. Guess I need to prod the kernel source